lib/openstudio-standards/standards/ashrae_90_1_prm/ashrae_90_1_prm.AirLoopHVAC.rb

```diff
@@ -81,36 +85,61 @@ def air_loop_hvac_apply_prm_baseline_fan_power(air_loop_hvac)
       end
     elsif system_type == 'PTAC' ||
           system_type == 'PTHP' ||
-          (system_type == 'Gas_Furnace' && !is_nmc) ||
-          (system_type == 'Electric_Furnace' && !is_nmc)
-      # Determine allowable fan power
-      allowable_power_w = 0.3
-    elsif (system_type == 'Gas_Furnace' && is_nmc) ||
-          (system_type == 'Electric_Furnace' && is_nmc)
+          system_type == 'Gas_Furnace' ||
+          system_type == 'Electric_Furnace'
+
       # Determine allowable fan power
-      allowable_power_w = 0.054
+      if !is_nmc
+        fan_efficacy_w_per_cfm = 0.3
+      else # is_nmc
+        fan_efficacy_w_per_cfm = 0.054
+      end
+
+      # Configuration is supply fan only
+      supply_fan_power_fraction = 1.0
```

Comment:
Only the NMC part of system 9 or 10 gets a reduced fan power, heating side is 0.3 W/cfm.